x/group/server/testsuite/genesis.go

```diff
@@ -0,0 +1,172 @@
+package testsuite
```

Comment:
I follow the existing structure and I just didn't want to put the genesis test in the already huge `suite.go` file. I don't mind adding `_test` suffix but then if you try to run the tests individually in `/testsuite` folder, it won't find any tests to run because the actual test with `suite.Run` is in `server_test.go` which is in a different package.

Reply:
Ah true. Hmm, okay let's keep it like this for now.